app/models/enterprise.rb

```diff
@@ -304,6 +304,10 @@ def instagram
     correct_instagram_url self[:instagram]
   end
 
+  def whatsapp_phone
+    correct_whatsapp_phone self[:whatsapp_phone]
+  end
```

Comment:
What if you didn't override this method but provided a `whatsapp_url` method which includes the `https://wa.me/` already. That way we avoid a potentially confusing override and bring the formatting and the place where the formatting is needed together. It makes sense as a unit. And all the views can just use that link without repeating the string interpolation.